cmake/targets/BuildMimalloc.cmake

```diff
@@ -19,6 +19,10 @@ set(MIMALLOC_CMAKE_ARGS
   -DMI_SKIP_COLLECT_ON_EXIT=ON
 )
 
+if(ENABLE_ASAN)
```

Comment:
you may also need to update `CompilerFlags.cmake` so other libraries get `-fsanitize=address`